docs/Users/Tutorials/External_Bears.rst

```diff
@@ -0,0 +1,529 @@
+External Bears
+==============
+
+Welcome. This tutorial will teach you how to use the
+``@external_bear_wrap`` decorator in order to write Bears in languages other
+than Python.
+
+.. note::
+
+  This tutorial assumes that you already know the basics. If you are new, please
+  refer to the
+  :doc:`Writing Bears<Writing_Bears>` section.
+
+  If you are planning to create a bear that uses an already existing tool (aka
+  linter), please refer to the
+  :doc:`Linter Bears<Linter_Bears>` section.
+
+Why is This Useful?
+-------------------
+
+coala is a great tool if you want to write static code analysis without having
+to worry about the interaction with the user. That makes it an excellent choice
+for researchers and other people who want to build new static analysis routines.
+Enabling developers to write external Bears means that they do not need to waste
```

Comment:
Put a newline before this line so that the reason and solution can be distinguished. Also I would trim down on the reasoning a little. Something like:

```
coala is great language independent static analysis tool, where users can write their own static analysis routines. Enabling users to write external ``bears`` means that they can write their own static analysis routine in their favourite language.
```